dora/core/server/worker/src/main/java/alluxio/worker/http/HttpServerHandler.java

```diff
@@ -61,12 +61,18 @@ public class HttpServerHandler extends SimpleChannelInboundHandler<HttpObject> {
 
   private final PagedService mPagedService;
 
+  private final FileSystem mFileSystem;
+
   /**
    * {@link HttpServerHandler} deals with HTTP requests received from Netty Channel.
    * @param pagedService the {@link PagedService} object provides page related RESTful API
    */
   public HttpServerHandler(PagedService pagedService) {
     mPagedService = pagedService;
+    FileSystemContext.FileSystemContextFactory factory =
+        new FileSystemContext.FileSystemContextFactory();
```

Comment:
Inject `FileSystemContext` instance instead of creating a new one.

Reply:
Updated.